contents/blog/posthog-vs-pendo.md

```diff
@@ -0,0 +1,651 @@
+            <td>Understand how organizations use your product</td>
+        </tr>
+        <tr>
+            <td><strong>Formulas</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Use custom formulas to calculate unique insights</td>
+        </tr>
+        <tr>
+            <td><strong>SQL access</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Write your own queries in SQL</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+- **Correlation analysis:** [Correlation analysis](/product-analytics) automatically highlights significant linking properties or events relevant to an insight. For example, you can find if users who complete a funnel are likely to be from a certain location, or have completed another event. In PostHog, correlation analysis is a standard part of analytics. 
+
+- **Formulas:** Formulas in PostHog enable you to create custom insights using your data. A simple example of a formula would be an equation to figure out a ratio or percentage (e.g. the percentage of user who completed two different events), though advanced formulas can use more elaborate functions, such as `COS` and `SIN`.
+
+- **SQL access:** While both PostHog and Pendo have ready-made insights and visualization types, only PostHog gives you unlimited access to your data by [writing your own SQL queries](/docs/product-analytics/hogql). This is ideal for data scientists, product managers, and engineers who want to perform advanced analysis on user data.
+
+<h4 className="mb-0">Discover what's possible with <span className="text-red">product analytics</span></h4>
+<TutorialsSlider slugs={[
+  "/tutorials/churn-rate",
+  "/tutorials/funnels",
+  "/tutorials/api-get-insights-persons",
+  "/tutorials/feature-retention",
+  "/tutorials/next-steps-after-installing"
+]} />
+
+### In-app messages and prompts
+Pendo is all about creating in-app messages, prompts, tooltips, measuring their impact on adoption and retention, and collecting feedback. But these aren't unique features – PostHog also supports in-app messages and interactive pop-ups through the use of [feature flag payloads](/docs/feature-flags/payloads) and [site apps](/tutorials/build-site-app).
+
+The main difference is, because Pendo is designed for less technical users, it offers an interface that is simpler, but less powerful. You can easily create tooltips, announcement pop-ups and polls in Pendo, but not other types of prompt such as a Calendly integration. 
+
+PostHog has ready-made apps for displaying pop-ups and tooltips, but the true power is in the ability to [build custom solutions](/tutorials/build-site-app).
```

Comment:
To be clear, the popups and tool tips app isn't entirely a site app. It is extra code added to the app + a feature flag. It could theoretically be a site app, but we don't really have a simple to set up popup/tooltip site app.